messaging/src/androidTest/java/io/lantern/messaging/MessagingTest.kt

```diff
@@ -123,9 +123,21 @@ class MessagingTest : BaseMessagingTest() {
                                 0,
                                 dog.db.listPaths(Schema.PATH_CONTACTS_BY_ACTIVITY.path("%")).count()
                             )
+                            val dogDbString = dog.db.dumpToString()
```

Comment:
Drive-by, this is just making absolutely sure that after we delete a contact, there's nothing left.